distro/run/qa/src/test/resources/ProductionConfigurationIntegrationTest_production.yml

```diff
@@ -0,0 +1 @@
+camunda.bpm.process-engine-name: production
```

Comment:
Do we know what database configuration will be used here now?
Should we define the in-memory db as well?

Reply:
Good hint. I think it works because of [this](https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/ProcessEngineConfiguration.java#L234-L237). (Not sure though)

We should define the standard in-memory db here.